roles/openshift_logging_elasticsearch/templates/es.j2

```diff
@@ -8,7 +8,7 @@ metadata:
     deployment: "{{deploy_name}}"
     logging-infra: "{{logging_component}}"
 spec:
-  replicas: {{replicas|default(1)}}
+  replicas: {{es_replicas|default(1)}}
```

Comment:
eventually with statefulsets or if you are using host mounts for storage you could just bump up the replicas instead of having multiple DC